src/platform/win32localfilesystem.cpp

```diff
@@ -29,11 +30,19 @@
 
 File *Win32LocalFileSystem::Open_File(const char *filename, int mode)
 {
-    if (strlen(filename) <= 0) {
+    using Win32BufferedFile = Thyme::Win32BufferedFile;
+
+    if (filename == nullptr || *filename == '\0') {
```

Comment:
I was so free and simplified this check here. There is no need to run an strlen if it just needs to know if first character is set. Also checks for null now as well.